.pre-commit-config.yaml

```diff
@@ -43,6 +42,12 @@ repos:
   rev: v2.16.3
   hooks:
   - id: pylint
+    args:
+    - '--load-plugins=pylint_django'
```

Comment:
It's best to list plugins in the config so when pylint is executed by other tools, it's picked up.

Reply:
I removed `--load-plugins=pylint_django` in `pre-commit-config.yaml` and kept in `pylinrc`:
` load-plugins=pylint_django`

Reply:
Marking this as resolved.

Reply:
Unmarking. Not yet implemented.

Reply:
Already implemented